test.sh

```diff
@@ -134,8 +134,8 @@ function verify() {
 function expected_errors() {
   local instance_file="$1"
   local type
-  type=$(yq -r '.properties | type' "$instance_file")
-  if [ "$type" != "object" ]; then
+  type=$(yq -r '.properties | type' "$instance_file" | sed 's/!!//')
+  if [[ "$type" != "object" && "$type" != "map" ]]; then
```

Comment:
I don't understand this change. What is a `"map"` and when would `yq`/`jq` output that type?

Reply:
I was getting YAML type tags `!!map` (Dictionary) instead of JSON type `object`. I couldn't figure out why you were getting different behaviour.

Locally installed yq version on Mac:
```
$ brew install yq
$ yq --version
yq (https://github.com/mikefarah/yq/) version v4.40.5
```

Reply:
Mine's older... I wonder if that makes a difference.

```sh
$ yq --version
yq 3.1.0
$ echo '{}' | yq type
"object"
```

Reply:
Done